include/libtorrent/kademlia/dht_storage.hpp

```diff
@@ -88,6 +88,14 @@ namespace dht
 		virtual size_t num_peers() const = 0;
 #endif
 
+		// PENDING REVIEW
+		// This method notifies the list of all node's ids of each
```

Comment:
I'm a bit of a pedantic when it comes to terminology. and in C++ it's "member function" not "method"